Create covid-education.md (#401)
medic/cht-docs: content/en/apps/examples/covid-education.md

---
title: "COVID-19 Education and Training for CHWs"
linkTitle: "COVID-19 Education"
weight: 
description: >

>  Education and training for Community Health Workers to combat COVID-19.

Reviewer: Use "address" or "mitigate" instead of "combat", and remove period from descriptions. I couldn't find reference to that in the doc style guide (yet!), but couldn't find another description with a period.

```suggestion
 Education and training for Community Health Workers to address COVID-19
```

keywords: training onboarding 
relatedContent: >
  apps/examples/training
  apps/examples/phc-covid
  apps/concepts/access/#magic-links-for-logging-in
---

The COVID-19 pandemic has created unique challenges to providing in-person Community Health Worker training. To support CHWs, three learning modules were created to rapidly and remotely train them on COVID-19 safety protocols and patient care. Through CHT app and SMS deployments, Supervisors can now train CHWs on care workflows without being physically present. This example covers the following learning modules:

* Health safety protocols for preventing the spread of COVID-19
* Preventing the dissemination of misinformation about COVID-19
* Recognizing COVID-19 and caring for patients with suspected infection

## Problem Being Addressed

> Now, more than ever, it is important for Community Health Workers to stay safe and serve their communities by understanding effective health protocols for preventing the spread of COVID-19. Training would traditionally be done in person, however, due to public health concerns, travel and gatherings of people are limited. Health systems have been forced to quickly adapt to deliver training content in a safely distanced, timely, and effective manner.  

Reviewer: ```suggestion
It is important for Community Health Workers to stay safe and serve their communities by understanding effective health protocols for preventing the spread of COVID-19. Training would traditionally be done in person, however, due to public health concerns, travel and gatherings of people are limited. Health systems have been forced to quickly adapt to deliver training content in a safely distanced, timely, and effective manner.  
```

## Solution Overview

> The CHT provides a flexible way to deliver timely COVID-19 remote training and education to CHWs. In collaboration with [Stanford's Digital Medic Team](https://digitalmedic.stanford.edu/), and vetted by a team of health experts at Stanford, a form-based application was designed to deliver three remote learning modules. Utilizing tasks to order the modules, CHWs can progress through training at their own pace. The training modules:

Reviewer: I think "form-based" will be confusing in the context of the doc site, reworded accordingly:

```suggestion
The CHT provides a flexible way to deliver timely COVID-19 remote training and education to CHWs. In collaboration with [Stanford's Digital Medic Team](https://digitalmedic.stanford.edu/), and vetted by a team of health experts at Stanford, three remote learning modules were designed. CHWs can progress through training at their own pace, and receive tasks in their CHT app to encourage them to complete the modules. The training modules:
```

* Build on verifiable health safety guidance from the CDC and WHO
* Can be deployed using the CHT app or SMS workflows 
* Capture assessment and understanding of CHW knowledge through quizzes, which can be tracked by Supervisors

### Users and Hierarchy Example

| User                             | Location                               | Devices                             | Role                                                                                                                                                                                                                                              |
| :------------------------------- | :------------------------------------- | :---------------------------------- | :------------------------------------------------------------------------------------------------- |
| National Officials and County Teams  | Central and district offices           | Desktop, laptop and smartphones     | Monitor central analytics regarding onboarding and training completion. Revisit content where needed.           |

> | Sub-County Teams       | Facilities or local offices            | Tablet or smartphone                | Review CHW progress on aggregate. Support CHW supervisors and monitor location specific analytics.            |

Reviewer 2: ```suggestion
| Sub-County Teams       | Facilities or local offices            | Tablet or smartphone                | Review CHW progress on aggregate. Support CHW supervisors and monitor location-specific analytics.            |
```

| CHW Supervisors                  | Community level, based at facilities   | Smartphones and personal phones     | Verify CHW onboarding and training completion. Follow-up with those who have not completed training. Complete their own onboarding and training tasks.                                    |
| CHWs and Volunteers              | Community level                        | Feature phones and some smartphones | Complete onboarding and training via SMS or App. Provide feedback and ask for support where needed.         |

## Workflow Example

> This demo illustrates how a CHT workflow can be easily adapted to integrate a sequence of learning modules through the usage of tasks. These modules are designed to be delivered sequentially to promote engagement and absorption of content. Learning can be tracked across CHWs to inform opportunities to provide additional support. The sequence can also be modified to interact with the CHW through SMS, utilizing 'question and reply' text-based workflows.

Reviewer: A bit of a personal preference, but the word "utilizing" is less approachable to readers.

```suggestion
This demo illustrates how a CHT workflow can be easily adapted to integrate a sequence of learning modules through the usage of tasks. These modules are designed to be delivered sequentially to promote engagement and absorption of content. Learning can be tracked across CHWs to inform opportunities to provide additional support. The sequence can also be modified to interact with the CHW through SMS, using question-and-answer text-based workflows.
```

<br />

{{< youtube pFEFIY_SA7M >}}

<br />

> For more detailed information and to download detailed content, please reference the documentation for both the [CHT application](https://docs.google.com/document/d/1Vd6x_WwS-S_sekRQnvW1UAS_rWaTnaawITfeXgX6Sdg) and the [SMS and Messaging workflow](https://docs.google.com/document/d/1fPqZhl5gYoW_UJx6E3EWngWcICOH8mfmslo4r5JqmE0). These showcase the content, key considerations for adapting the content, and the technical implementation. 

Reviewer 2: ```suggestion
For more detailed information and to download detailed content, please reference the documentation for both the [CHT application](https://docs.google.com/document/d/1Vd6x_WwS-S_sekRQnvW1UAS_rWaTnaawITfeXgX6Sdg) and the [SMS and Messaging workflow](https://docs.google.com/document/d/1fPqZhl5gYoW_UJx6E3EWngWcICOH8mfmslo4r5JqmE0). These showcase the content, key considerations for adapting it, and the technical implementation. 
```

> For more detailed information and to download detailed content, please reference the documentation for both the [CHT application](https://docs.google.com/document/d/1Vd6x_WwS-S_sekRQnvW1UAS_rWaTnaawITfeXgX6Sdg) and the [SMS and Messaging workflow](https://docs.google.com/document/d/1fPqZhl5gYoW_UJx6E3EWngWcICOH8mfmslo4r5JqmE0). These showcase the content, key considerations for adapting the content, and the technical implementation. 

Reviewer: The first link is undershared (can't access) and the second is overshared (anyone with link can edit). Can access be updated accordingly? Also, where appropriate can the content be included here, or link to a published paper?

Author: I discussed with Helen and David (original authors / squad leads) and decided to only showcase the SMS document for additional information (access corrected). Also updated the text to better align with the other examples.

Reviewer: Ok, I still see both links in the [newest version](https://github.com/medic/cht-docs/blob/1d379cae6d716370fe7e8f0ad6639aaa6c72f972/content/en/apps/examples/covid-education.md#workflow-example), and can't access the [CHT application doc](https://docs.google.com/document/d/1Vd6x_WwS-S_sekRQnvW1UAS_rWaTnaawITfeXgX6Sdg).
